homeworks/dmytro.kondaurov_DmitriyKondaurov/3-async/main.js

```diff
@@ -0,0 +1,82 @@
+const fieldForPosts = document.body.querySelector('[data-main-box]');
+let listOfPosts = [];
+const postFilter = document.body.querySelector('[data-filter]');
+const postSort = document.body.querySelector('[data-sort]');
+let template = document.querySelector('[data-post-template]');
+
+const hideLoaderGifImage = () => {
```

Comment:
```suggestion
const hideLoader = () => {
```
Достатньо такого. `hideLoaderGifImage ` має забагато деталей в назві. Що гіфка заміниться на svg чи на теги та стилі? Потрібно буде ходити змінювати назви. Достатньо знати лише того що це  лодер і ця функція його ховає

Reply:
done